ecommerce/migrations/0005_application_multiple_orders.py

```diff
@@ -1,26 +0,0 @@
-# Generated by Django 2.2.10 on 2020-05-11 17:49
```

Comment:
Undelete this and increment the number to 00006 for the next migration?  The app exits with an error for me because this migration has already been applied.

Reply:
:thinking: not sure how that happened... probably a mistaken delete

Reply:
Working now, got a wire transfer to show up in the application statement!